### PR TITLE
feat: Adds an option to disable iframeAPI.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

**Review by jallamsetty1 — APPROVED** [review]

File: react/features/app/actions.web.ts [hunk 1]
```diff
@@ -129,6 +129,14 @@ export function appNavigate(uri?: string) {
 
         dispatch(setLocationURL(locationURL));
         dispatch(setConfig(config));
+
+        if (Number(API_ID) === API_ID && getState()['features/base/config'].disableIframeAPI) {
```
Comment (saghul) [inline react/features/app/actions.web.ts:133]: Why the check for API_ID? We have some isInIframe helpers, wouldn't that be more explicit?

**Review by saghul — COMMENTED** [review]
LGTM with a question

File: react/features/app/actions.web.ts (hunk 1 above)
Comment (damencho) [reply react/features/app/actions.web.ts:133]: Cause the same check exists in the same file. I was not aware of helpers. I will check it out.

Comment (damencho) [reply react/features/app/actions.web.ts]: Done.

**Review by jallamsetty1 — APPROVED** [review]

**Pull request merged**